test/e2e/config/saucelabs.conf.js

```diff
@@ -0,0 +1,154 @@
+const chai = require('chai');
```

Comment:
maybe call this cross browser config rather than sauselab

Reply:
sure @alectronic0  will update now